data/interfaces.txt

```diff
@@ -118,16 +118,14 @@ color "map orbits fleet destination" 1. 1. 1. 1.
 
 
 interface "map detail panel"
-	value "planet starting Y" 90
-	value "planet max bottom Y" 340
-	value "relative trade Y after planet" 135
-	value "planet scroll speed" 25
+	value "planet panel height" 460
 	value "starting X" 100
 	value "starting Y" 45
-	value "government top Y" 45
+	value "government Y" 800
```

Comment:
800 sends it off my screen.
```suggestion
	value "government Y" 522
```
![image](https://user-images.githubusercontent.com/17688683/183020245-df7407e4-fd05-4839-bfd3-cc833a998961.png)
While this is more lined up, a bunch of the click zones in these two panels feel misaligned.

Reply:
Are the clickzones okay now?